conf/eddie.config

```diff
@@ -0,0 +1,50 @@
+//Profile config names for nf-core/configs
+params {
+  config_profile_description = 'University of Edinburgh (eddie) cluster profile provided by nf-core/configs.'
+  config_profile_contact = 'Alison Meynert (@ameynert)'
+  config_profile_url = 'https://www.ed.ac.uk/information-services/research-support/research-computing/ecdf/high-performance-computing'
+}
+
+executor {
+  name = "sge"
+  queueSize = "100"
+}
+
+process {
+  clusterOptions = { task.memory ? "-l h_vmem=${task.memory.bytes/task.cpus}" : null }
```

Comment:
This could give decimals - is that ok? May be worth rounding if in doubt.

Reply:
(Also when I've done this in the past I have subtracted a little for good measure)

Reply:
ah yeah, as you are doing in the pipeline specific configs I see 👍🏻 Could just do here for all processes and skip the need for pipeline specific configs?

Reply:
It's working ok so far without rounding, so will not attempt that. It's only the Java processes that require the overhead, and we sometimes need to tweak those a little further so prefer to keep the pipeline-specific configs.